feat: Add a param isFIPSEnabled (#598)
DataDog/serverless-plugin-datadog: README.md

| `enableStepFunctionsTracing`    | Enable automatic subscription of the Datadog Forwarder to Step Function log groups and Step Functions tracing. If no Step Function log groups are configured, then they are automatically created. Requires setting `forwarderArn`. Defaults to `false`.                                                                                                                                                                  |
| `propagateUpstreamTrace` | When set to `true`, downstream Stepfunction invocation traces merge with upstream Stepfunction invocations. Defaults to `false`. |
| `redirectHandlers`    | Optionally disable handler redirection if set to `false`. This should only be set to `false` when APM is fully disabled. Defaults to `true`.                                                                                                                                                                  |

> | `enableFIPS`    |  When set to `true`, a FIPS-compliant lambda extension layer will be used. Only works if `addExtension` is `true` and `site` is `ddog-gov.com`. Defaults to `false`.                                                                                                                                                                  |

Reviewer: ```suggestion
| `enableFIPS`    |  When set to `true`, a FIPS-compliant Lambda extension layer is used. This only works if `addExtension` is `true`, and `site` is `ddog-gov.com`. Defaults to `false`.                                                                                                                                                                  |
```

> | `enableFIPS`    |  When set to `true`, a FIPS-compliant Lambda extension layer is used. This only works if `addExtension` is `true`, and `site` is `ddog-gov.com`. Defaults to `false`.                                                                                                                                                                  |

Reviewer 2: We should default this prop to `true` when we see that `site` is `ddog-gov.com`.

Author: Addressed
